wremnants/plot_tools.py

```diff
@@ -397,6 +393,22 @@ def makePlotWithRatioToRef(
     if x_ticks_ndp: ax2.xaxis.set_major_formatter(StrMethodFormatter('{x:.' + str(x_ticks_ndp) + 'f}'))
     return fig
 
+def extendEdgesByFlow(href, bin_flow_width=0.02):
```

Comment:
There is actually an option in mpl hep to plot the overflow. I'm not 100% sure if what they have suits your purposes, but the option is "flow", documented here: https://mplhep.readthedocs.io/en/latest/api.html

Reply:
I recently changed this to "none" in my WIP PR because it was annoying me that it now causes those \\ marks on the plot edges otherwise, but it could be configurable

Reply:
I tried using the "flow" option in the past but it didn't work.

Reply:
ok not prob then